src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -8,8 +8,7 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e",
-    "precommit": "pretty-quick --staged"
```

Comment:
Keeping 'precommit' is useful if you do not have an IDE plugin to prettify code.

Reply:
Could we add a new command like `makePretty` that will not run on the staged files? That would be nice and helpful as I can observe the changes `pretty-quick` does before I commit. This way it's not automatically triggered, but it's the same with linting and you have the chance to observe the changes.